hw/dv/dpi/usbdpi/usbdpi.c

```diff
@@ -543,6 +588,46 @@ void pollRX(struct usbdpi_ctx *ctx, int sendHi, int nakData) {
   }
 }
 
+int set_driving(struct usbdpi_ctx *ctx, int d2p, int newval) {
+  if (d2p & D2P_DNPU) {
+    if (d2p & D2P_TXMODE_SE) {
+      return ((ctx->driving & P2D_SENSE) |
```

Comment:
Throughout: do not write `return (...);`. The parentheses are always redundant.

Reply:
Done, several places. Although I worry this will now look messier if the code gets auto indented.